doc/sphinx/source/utils/RTW/add_a_recipe.rst

```diff
@@ -26,48 +26,80 @@ How to add a recipe to the |RTW|
      [...]
      YYYY-MM-DD HH:MM:SS:sss UTC [12345] INFO    Run was successful
 
-#. Add the recipe to the ``[task parameters]`` section in the
-   ``esmvaltool/utils/recipe_test_workflow/flow.cylc`` file.
+.. _run_a_recipe_in_the_rtw:
 
-   .. hint::
-      If the recipe takes less than 10 minutes to run then it should be added
-      to the ``fast`` option. Recipes that take longer than ten minutes should
-      be added to the ``medium`` option.
+Run a Recipe in the |RTW|
```

Comment:
I feel this heading should be "Add the recipe to the RTW", but that's quite similar to the title of this documentation page! 😱

Reply:
@chrisbillowsMO and I worked together on the documentation and now have a lovely set of headings! a353bf7 🎉